docs/source/DeveloperGuide/AddingNewComponents/DevelopingCommands/index.rst

```diff
@@ -2,6 +2,10 @@
 Developing Commands
 ======================================
 
+.. warning::
+    This instructions here demonstrate how to support both the legacy (Python 2) and future (Python3) installations of DIRAC.
+    If only having Python 3 support is acceptable, the requirement for scripts to be in the the *scripts* directory of their parent system will be removed and the only requirement will be for the function to be decorated with the ``@DIRACScript()`` decorator.
```

Comment:
I actually am happy with having th scritps in the script directory of their parent system. What's wrong with that ?

Reply:
It then becomes a convention as opposed to a requirement so I think this is valid to state this instead of claiming it's a firm requirement, even if we keep requiring it in vanilla DIRAC.

Some scripts are so trivial that it might be more natural to add the wrapper to a function elsewhere in cases like `dirac_platform.py` or the `ScriptExecutors` pattern that is used in LHCbDIRAC. Ultimately I think we should punt on it until we overhaul the DIRAC CLI in a backwards incompatible way.

Reply:
yeah maybe. I just wouldn't have said even a word about it now :D